app/models/job_invocation.rb

```diff
@@ -24,6 +24,13 @@ class JobInvocation < ActiveRecord::Base
 
   scoped_search :on => [:job_name], :complete_value => true
 
+  scoped_search :in => :last_task, :on => :started_at, :rename => 'last_task.started_at'
+  scoped_search :in => :last_task, :on => :ended_at, :rename => 'last_task.ended_at'
```

Comment:
Can we just rename these to started_at/ended_at? It looks more user-friendly in the search bar